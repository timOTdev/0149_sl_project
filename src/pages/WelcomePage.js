import React from 'react'
import { Greeting, Form } from '../components/'
import styled from 'styled-components'

const WelcomePage = () => (
  <Div>
    <Greeting />
    <Form />
  </Div>
)

const Div = styled.div`
  display: flex;
  margin: 0 auto;
  @media (max-width: 320px) {
    flex-direction: column;
  }
`

export default WelcomePage
