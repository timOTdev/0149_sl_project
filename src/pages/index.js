// index for all page files
import WelcomePage from './WelcomePage'

export {
  WelcomePage
}
