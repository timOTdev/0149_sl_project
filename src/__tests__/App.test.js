import React from 'react';
import { render, cleanup } from 'react-testing-library'
import 'jest-dom/extend-expect'
import { WelcomePage } from '../pages'

afterEach(cleanup)

describe('-- App.js --', () => {
  it('renders', () => {
    const { container } = render(<WelcomePage />)
    expect(container).toMatchSnapshot()
  })
})
