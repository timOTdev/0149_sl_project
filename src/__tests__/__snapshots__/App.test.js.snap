// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`-- App.js -- renders 1`] = `
<div>
  <div
    class="sc-gZMcBi iKHraN"
  >
    <section
      class="sc-bdVaJa usAgg"
    >
      <svg
        class="sc-bwzfXH dmdIqG"
        data-testid="svg-element"
      >
        GiantRobotLTD_Logo.svg
      </svg>
      <h1
        class="sc-htpNat iuIPbH"
        data-testid="h1tag"
      >
        Welcome
      </h1>
      <p
        class="sc-bxivhb kfOGlP"
        data-testid="ptag"
      >
        Please tell us a bit about yourself to get started.
      </p>
    </section>
    <section
      class="sc-ifAKCX PvtxZ"
    >
      <form
        class="sc-EHOje KQgPY"
      >
        <label
          class="sc-bZQynM hLMmSv"
          data-testid="label1"
          for="firstName"
        >
          FIRST NAME
        </label>
        <input
          class="sc-gzVnrw HcCoo"
          data-testid="input1"
          name="firstName"
          required=""
          type="text"
          value=""
        />
        <label
          class="sc-bZQynM hLMmSv"
          data-testid="label2"
          for="lastName"
        >
          LAST NAME
        </label>
        <input
          class="sc-gzVnrw HcCoo"
          data-testid="input2"
          name="lastName"
          required=""
          type="text"
          value=""
        />
        <label
          class="sc-bZQynM hLMmSv"
          data-testid="label3"
          for="address1"
        >
          ADDRESS
        </label>
        <input
          class="sc-gzVnrw HcCoo"
          data-testid="input3"
          name="address1"
          required=""
          type="text"
          value=""
        />
        <label
          class="sc-bZQynM hLMmSv"
          data-testid="label4"
          for="address2"
        >
          ADDRESS 2 (OPTIONAL)
        </label>
        <input
          class="sc-htoDjs gehhcT"
          name="address2"
          placeholder=" "
          type="text"
          value=""
        />
        <button
          class="sc-dnqmqq bpbtYo"
          data-testid="button"
          type="submit"
        >
          Next
          <svg
            class="sc-iwsKbI eXVGwH"
          >
            White_Arrow.svg
          </svg>
        </button>
      </form>
    </section>
  </div>
</div>
`;
